src/openrct2/platform/Platform.h

```diff
@@ -40,20 +40,14 @@ enum class SPECIAL_FOLDER
     RCT2_DISCORD,
 };
 
-struct rct2_date
+struct PODDate
```

Comment:
POD?

Reply:
Plain old data. I couldn't think of something better to split it from the existing Date class we have.

Reply:
RealWorldDate?